### ISSUE TITLE
Bug/Lack Of Functionality: Document Signature Link Handling

### ISSUE THREAD

**Issue by JFoxUK** [qwilio/qwil-salesforce-app#10]
When a staff member sends a client a document to sign, the message appears in the chat with no hyperlink or away to action the signature. There is also no instruction on what to do.
![PHOTO-2024-07-15-08-49-11](https://github.com/user-attachments/assets/c07fe3f5-31b1-416f-a2ff-3d35e0aff6d3)

**Comment by shawnchin** [comment]
Yup. The current SDK only supports a subset of message types (there are LOTS). We well add support for different message types over time based on demand.

Will keep this request in mind the next time we prioritise the backlog.
